### ISSUE TITLE
paused in transmission -> gone in transgui

### ISSUE THREAD

**Issue by robertdahlem** [leonsoft-kras/transmisson-remote-gui#907]
I'm using transmissiond 2.84 on a Synology NAS. It has it's own Web GUI, which is a bit clumsy, so I prefer Transmission Remote GUI. The transmissiond Web GUI has a pause button where I can stop torrents from downloading or seeding without deleting them. The status in the database table download_queue changes from 8 to 3 then, the pid from something positive to -1. The second I do this the torrent becomes invisible in transgui.

transmissiond still seems to report paused torrents because I can see them in the Synodroid Android app (as paused).

Also: if I stop a torrent from transgui (which I expect to be the same as pause), it becomes invisible in transgui, but nothing changes in the transmissiond Web GUI nor the database table.

**Comment by leonsoft-kras** [comment]
Аttach screenshots because can not understand anything.
You can add comments and explanations.

**Comment by moonman** [comment]
Did you apply any filters in transgui? Maybe it is set up to only show "active" torrents or only "downloading"?

**Comment by robertdahlem** [comment]
I have these files in transmissiond:
![hardcopy1](https://cloud.githubusercontent.com/assets/5524999/15098574/91455b9a-1540-11e6-8ebd-22bb7a371aa3.jpg)

In transgui it looks like this:
![hardcopy2](https://cloud.githubusercontent.com/assets/5524999/15098582/a8ee5436-1540-11e6-80d5-4abec46b0b99.jpg)

Now I pause the first file in transmissiond:
![hardcopy3](https://cloud.githubusercontent.com/assets/5524999/15098586/b8e73786-1540-11e6-941c-2dd3bd1be948.jpg)
Notice Status is "Paused"

The same moment the file vanishes from transgui:
![hardcopy4](https://cloud.githubusercontent.com/assets/5524999/15098589/c4e58f4c-1540-11e6-9c74-59ee6e5799c8.jpg)

When I restart the torrent in transmissiond, it reappears in transgui.

**Comment by leonsoft-kras** [comment]
Yes, it is seen that the total number of torrents has changed from 452 to 451.
It is unclear why the TRG has received such information. Not 100% sure, but TRG does not filter the total number of torrents in the list.

I can recommend is to change (update) version with transmissiond. 
maybe it will help.

**Comment by robertdahlem** [comment]
As I wrote before: the Android app "Synodroid" (another GUI for transmissiond) shows these torrents as "paused". This is why I consider this a bug in TRG.

**Comment by leonsoft-kras** [comment]
Thousands of TRG people  are still not notice this problem. All put on pause and everything is fine.

I do not know yet what the problem is. Especially since I'm not the author of the program. I try to see it, but the result is not guaranteed. I do not know how to simulate your problem.

**Comment by cfpp2p** [comment]
Looks like a very unusual problem. I still think it could be a problem with the daemon. Synology NAS has been notoriously buggy with the daemon. I would try a different version of the daemon on the same box to see if anything different happens. Also see if possibly there is more than one instance of the daemon running on the box ( ps -aux | grep transmission ). What happens if torrent(s) are stopped using the web-client? What about older versions of TRG like v3.2 or 4.1? Hopefully it won't be too difficult to narrow down the possibilities a little.

**Comment by typhoon71** [comment]
Just chipping in, I was searching for something here and saw this post.

I tested Transmission-daemon (Raspbian Jessie on a Raspberry Pi) and TRG 5.40, and when I pause a torrent using transmission web interface it is still visible in TRD, it doesn't became invisible. 

This makes me think Synology build of the daemon is buggy, as cfpp2p suggested.
